src/components/implementation/no_interface/cFE_booter/osfiles.c

```diff
@@ -220,27 +285,22 @@ int32 OS_rmdir(const char *path)
 /*
  * Makes a file system
 */
-int32 OS_mkfs(char *address,char *devname, char *volname,
+int32 OS_mkfs(char *address, char *devname, char *volname,
                                 uint32 blocksize, uint32 numblocks)
 {
-    uint32 ret = 0;
     if (address)                return OS_FS_ERR_INVALID_POINTER;
     if (!devname || !volname)   return OS_FS_ERR_INVALID_POINTER;
     if (strlen(devname) >= OS_FS_DEV_NAME_LEN || strlen(volname) >= OS_FS_VOL_NAME_LEN)
-                                return OS_FS_ERROR;
-    ret = newfs_init(devname, volname, blocksize, numblocks);
-    if (ret != OS_FS_SUCCESS)   return ret;
-    ret = tar_parse();
-    if (ret != OS_FS_SUCCESS)   return ret;
-    return OS_FS_SUCCESS;
+                                return OS_FS_ERR_PATH_TOO_LONG;
+    return newfs_init(devname, volname, blocksize, numblocks);
```

Comment:
It inits a new fs.  by popular demand I think I will change around the naming to this and others